apps/video-importer/build-scripts/build-cross-platform.sh

```
@@ -0,0 +1,90 @@
+cd "$PROJECT_DIR"
+echo "Generating SEA blob..."
+node --experimental-sea-config sea-config.json
+
+# Create output directory
+mkdir -p "$DIST_DIR"
+
+# Function to download Node.js binary if not exists
+download_node_binary() {
+    local platform=$1
+    local arch=$2
+    local ext=$3
```

Shellcheck (0.10.0)

[warning] ext appears unused. Verify use (or export if used externally).

(SC2034)